src/components/thumbnail/right-nav-tab-contents.tsx

```diff
@@ -5,7 +5,9 @@ import { BaseComponent, IBaseProps } from "../base";
 import { DocumentDragKey, DocumentModelType } from "../../models/document/document";
 import { ENavTabSectionType, ERightNavTab, navTabSectionId, NavTabSectionModelType } from "../../models/view/right-nav";
 import { DocumentsSection } from "./documents-section";
-
+import { FourUpComponent } from "../four-up";
+import { GroupVirtualDocument } from "../../models/document/group-vritual-document";
+import { EPanelId } from "../../clue/components/clue-app-content";
```

Comment:
This reference to CLUE-specific code from common code is questionable. Need to figure out a more application-neutral way of accomplishing this. (If not now then soon, before we forget.) Of the changes I'm proposing, this is the only one that seems non-straightforward. One solution would be to move the tab definitions into app-config, but at this point that can wait for a subsequent cleanup.